src/NonStrippingCreditCard.php

```diff
@@ -28,4 +28,104 @@ public function setNumber($value)
          */
         return $this->setParameter('number', $value);
     }
+
+    /**
+     * Sets the payment instrument name.
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setPaymentInstrumentName($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('paymentInstrumentName', $value);
+    }
+
+    /**
+     * Gets the payment instrument name.
+     *
+     * @return string|null
+     */
+    public function getPaymentInstrumentName()
+    {
+        return $this->getParameter('paymentInstrumentName');
+    }
+
+    /**
+     * Sets the payment network.
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setPaymentNetwork($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('paymentNetwork', $value);
+    }
+
+    /**
+     * Gets the payment network.
+     *
+     * @return string|null
+     */
+    public function getPaymentNetwork()
+    {
+        return $this->getParameter('paymentNetwork');
+    }
+
+    /**
+     * Sets Apple Pay's transaction reference.
+     * It's Apple Pay's reference to a transaction.
```

Comment:
this line may be redundant given the one above... probably more helpful to mention that apple calls this a transactionIdentifier